docs/src/Master_Documentation.adoc

```diff
@@ -319,12 +321,10 @@ include::code/rs274.adoc[]
 
 :leveloffset: 0
 
-include::common/glossary.adoc[]
+//include::common/glossary.adoc[]
```

Comment:
Shouldn't this be included again?

Furthermore we have two headings "Configuration", one in line 117 and one in 135. I think one is too much?

Reply:
Sorry, yes, this is an oversight.

Reply:
And also the Master_Documentation is currently in `docs/src` instead of `docs`. But I assume this isn't finished yet...

Reply:
I am not aware to have moved that.

Reply:
I only wanted to mention that it finally should be in `docs`